scripts/ispc-disp

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
```

Comment:
1. License. you can use short pointer to the license, like this one: https://github.com/ospray/ospray/blob/master/scripts/files-to-md5.sh#L3
2. This is new file without a single line of comments or general description of what this file is doing.

Reply:
Done